.travis.yml

```diff
@@ -74,4 +74,4 @@ script:
   - ngx-build $NGINX_VERSION --with-ipv6 --with-http_realip_module --with-http_ssl_module --with-cc-opt="-I$OPENSSL_INC" --with-ld-opt="-L$OPENSSL_LIB -Wl,-rpath,$OPENSSL_LIB" --add-module=../ndk-nginx-module --add-module=../lua-nginx-module --add-module=../lua-upstream-nginx-module --add-module=../echo-nginx-module --with-debug > build.log 2>&1 || (cat build.log && exit 1)
   - nginx -V
   - ldd `which nginx`|grep -E 'luajit|ssl|pcre'
-  - prove -r t
+  - find t -maxdepth 1 -name "*.t" -type f | sort | xargs -I{} sh -c 'sleep 0.1; prove "{}" || (cat t/servroot/logs/error.log && exit 1)'
```

Comment:
why change this ?

Reply:
Because otherwise the test just fails and there is no way to see, why. The error.log file gets overridden with every new test case and you can only see the last tests result, if even that.
So I changed it, that the pipeline shows the contents of error.log whenever there is a failing test.
Locally the tests worked, in Travis they didn't and this was the only way to see, why.

Reply:
We can revert that or change the maxdepth whatever. Maybe there is even a less hacky way to do that. Anyway, I suggest we keep a way to see the contents of a failing tests error.log.